iosApp/iosApp/Utils/Modifiers/FavoritesModifier.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  FavoritesModifier.swift
+//  iosApp
+//
+//  Created by esimon on 9/4/25.
+//  Copyright © 2025 MBTA. All rights reserved.
+//
+
+import Shared
+import SwiftUI
+
+class LoadedFavorites {
+    static var last = Favorites(routeStopDirection: [])
+}
+
+struct FavoritesModifier: ViewModifier {
+    var favoritesRepository: IFavoritesRepository = RepositoryDI().favorites
+    @Binding var favorites: Favorites
+    @Binding var awaitingUpdate: Bool
+
+    func loadFavorites() {
+        guard awaitingUpdate else { return }
```

Comment:
I think the fact that the `.onAppear()` won’t load the favorites if `awaitingUpdate` is `false` is counterintuitive and a little risky, and I’d rather move this check into the `onChange` in the short term.